src/recordlinker/schemas/link.py

```diff
@@ -41,10 +41,29 @@ class LinkResult(pydantic.BaseModel):
     person_reference_id: uuid.UUID = pydantic.Field(
         description="The identifier for a person that the patient may be linked to."
     )
-
-    belongingness_ratio: typing.Annotated[float, pydantic.Field(ge=0, le=1)] = pydantic.Field(
-        description="The proportion of patient records matched in this person cluster ("
-        "between 0 and 1.0)."
+    accumulated_points: float = pydantic.Field(
+        description="The median number of log-odds points accumulated by the blocked patients "
+        "belonging to this Person Cluster when compared to an incoming record."
+    )
+    pass_label: str = pydantic.Field(
+        description="The label associated with the algorithm pass that generated this "
+        "link result."
+    )
+    rms: typing.Annotated[float, pydantic.Field(ge=0, le=1)] = pydantic.Field(
```

Comment:
I just did a quick scan of the API to see what kinds of labels we have for schema properties.  Outside of `kwargs` on the algorithm pass (which is in the process of being removed), I didn't see any other places where we use acronyms for property names.

This doesn't have to be done now, but before we release v25.5.0, I think we should consider replacing these (rms, mmt, cmt) acronyms with full names.